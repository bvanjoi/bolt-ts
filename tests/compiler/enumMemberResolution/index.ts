// From `github.com/microsoft/TypeScript/blob/v5.9.2/tests/cases/compiler/enumMemberResolution.ts`, Apache-2.0 License

enum Position2 {
    IgnoreRulesSpecific = 0
}
var x = IgnoreRulesSpecific. // error
//~^ ERROR: Cannot find name 'IgnoreRulesSpecific'.
var y = 1;
//~^ ERROR: Identifier expected.
var z = Position2.IgnoreRulesSpecific; // no error
