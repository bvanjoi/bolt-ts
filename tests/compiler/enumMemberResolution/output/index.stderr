  × Identifier expected.
   ╭─[../../tests/compiler/enumMemberResolution/index.ts:8:1]
 8 │ var
   · ───
   ╰────

  × Cannot find name 'IgnoreRulesSpecific'.
   ╭─[../../tests/compiler/enumMemberResolution/index.ts:6:9]
 6 │ IgnoreRulesSpecific
   · ───────────────────
   ╰────
